src/CommonDBTM.php

```diff
@@ -1819,6 +1820,7 @@ protected function cleanLockeds()
             && $this->isDynamic()
             && (in_array('is_dynamic', $this->updates) || isset($this->input['is_dynamic'])
             && $this->input['is_dynamic'] == true)
+            && !isset($this->input['_replace'])
```

Comment:
What would be the consequence of testing `_replace_by` rather than `_replace` (to avoid changing the variable name)?